test.properties

```diff
@@ -878,14 +880,11 @@
         apps/portal-security-audit/portal-security-audit-storage-service,\
         apps/portal-security-audit/portal-security-audit-storage-test,\
         apps/portal-security-audit/portal-security-audit-web,\
-        apps/portal-workflow/portal-workflow-kaleo-designer-api,\
-        apps/portal-workflow/portal-workflow-kaleo-designer-service,\
-        apps/portal-workflow/portal-workflow-kaleo-designer-test,\
-        apps/portal-workflow/portal-workflow-kaleo-designer-web,\
-        apps/portal-workflow/portal-workflow-kaleo-forms-api,\
-        apps/portal-workflow/portal-workflow-kaleo-forms-service,\
-        apps/portal-workflow/portal-workflow-kaleo-forms-test,\
-        apps/portal-workflow/portal-workflow-kaleo-forms-web,\
+        dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web,\
+        dxp/apps/portal-workflow/portal-workflow-kaleo-forms-api,\
+        dxp/apps/portal-workflow/portal-workflow-kaleo-forms-service,\
+        dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test,\
+        dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web,\
```

Comment:
@kyle-miho this doesn't look right. All of the apps in this list are dxp https://github.com/liferay/liferay-portal/tree/master/modules/dxp/apps. I feel like no change is needed here, and `dxp/` is set elsewhere.

Reply:
Some of them are not dxp apps, such as portal-security-audit

Reply:
@jpince
        apps/portal-cache/portal-cache-ehcache-ee-provider,\
        apps/portal-cache/portal-cache-multiple,\
        apps/portal-rules-engine/portal-rules-engine-drools,\
        apps/portal-rules-engine/portal-rules-engine-drools-test,\
        apps/portal-scheduler/portal-scheduler-multiple,\
these are only in modules/apps
       wile the workflow ones for these submodules are only existing in dxp/apps. Although since these are all built together now, maybe   modules.private.dir.names= isnt a necessary property for any of these, since none of these are pulled from master-private anymore

Reply:
@kyle-miho you're right, this is a bit of a mixed list. Can you follow up with CII on your last statement? I suspect that CI previously used it for their custom script to pull in the private modules, so it may indeed no longer be needed.